### PR TITLE
Add deviceID to Playable for history entries

### PR DESCRIPTION
**Pull request by glensc**
This is needed to identify devices from play history:

```python
    history = movie.history()
    for h in history:
        print("- Watched: %s (%s) -> %s,%s" % (h.viewedAt, h.lastViewedAt, h.accountID, h.deviceID))
```

### CODE REVIEW

**Comment by glensc** [comment]
Also need some help, how to find Device based on that Id?

There's `plex.systemDevices()`, but it returns all devices, not the specific one I'm interested of.

Also, the offset is off by one when accessing as index, so need to use lookup;

I've come up with this:
```python
    devices = plex.systemDevices()
    history = movie.history()
    for h in history:
        # device = devices[h.deviceID] <- devices is not indexed by deviceId
        device = next(x for x in devices if x.id == h.deviceID)
```

File: plexapi/base.py [hunk 1]
```diff
@@ -538,6 +538,7 @@ def _loadData(self, data):
         self.session = self.findItems(data, etag='Session')                         # session
         self.viewedAt = utils.toDatetime(data.attrib.get('viewedAt'))               # history
         self.accountID = utils.cast(int, data.attrib.get('accountID'))              # history
+        self.deviceID = utils.cast(int, data.attrib.get('deviceID'))                # history
```
Comment (JonnyWong16) [inline plexapi/base.py:541]: Could you also add `accountID` and `deviceID` to the doc string?

**Review by JonnyWong16 — CHANGES_REQUESTED** [review]

**Comment by JonnyWong16** [comment]
Using `next()` will work.

I added convenience methods to retrieve the account or device by ID in #703.

File: plexapi/base.py (hunk 1 above)
Comment (glensc) [reply plexapi/base.py:543]: f6596a03cea3e69a1e1e93aa29141fe261d25720

**Comment by glensc** [comment]
> Using `next()` will work.
> 
> I added convenience methods to retrieve the account or device by ID in #703. The methods just use `next()` the same way.

Maybe (also) make the devices indeed by deviceId?

**Comment by JonnyWong16** [comment]
> Maybe (also) make the devices indeed by deviceId?

I don't know what you mean.

**Review by JonnyWong16 — APPROVED** [review]

**Pull request merged**